Automattic/wp-calypso
# Stats: Add a tooltip for annual highlights

#### Proposed Changes

* adding a tooltip to display detailed numbers when hovering the number section of a highlight card
* ~~enhancing annual highlights by adding arrows to navigate between years~~ moved to #71343

Before:
![Screenshot 2022-12-14 at 1 52 25 PM](https://user-images.githubusercontent.com/112354940/207477614-531d7a97-9dee-4dc5-aa51-e9be316cda57.png)

After:
![Screenshot 2022-12-20 at 2 17 55 PM](https://user-images.githubusercontent.com/112354940/208558820-20574b1f-8969-4b6a-ae13-9287cc021ce2.png)


#### Testing Instructions



* navigate to the `Insights` section of Stats
* hover over the card's numbers and verify that a tooltip shows up. When hovering a shortened number, the tooltip should show an accurate number, not the shortened version

#### Pre-merge Checklist





Related to #70206
